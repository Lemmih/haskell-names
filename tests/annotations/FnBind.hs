module FnBind where

f x y = x + y + z + t
  where
    -- NB: mutually recursive bindings
    z = x + t
    t = y + z
