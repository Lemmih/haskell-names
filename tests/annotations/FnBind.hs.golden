x        at  3:9 is a local value defined at 3:3
+        at 3:11 is not in scope
y        at 3:13 is a local value defined at 3:5
+        at 3:15 is not in scope
z        at 3:17 is a local value defined at 6:5
+        at 3:19 is not in scope
t        at 3:21 is a local value defined at 7:5
x        at  6:9 is a local value defined at 3:3
+        at 6:11 is not in scope
t        at 6:13 is a local value defined at 7:5
y        at  7:9 is a local value defined at 3:5
+        at 7:11 is not in scope
z        at 7:13 is a local value defined at 6:5
x        at 9:13 is a local value defined at 9:6
+        at 9:15 is not in scope
y        at 9:17 is a local value defined at 9:9
+        at 9:19 is not in scope
z        at 9:21 is a local value defined at 9:2
+        at 9:23 is not in scope
t        at 9:25 is a local value defined at 11:5
y        at 11:9 is a local value defined at 9:9
+        at 11:11 is not in scope
z        at 11:13 is a local value defined at 9:2
